docs/source/spack_build.rst

```diff
@@ -0,0 +1,120 @@
+.. spack_build:
+
+Building with Spack
+===================
+
+Spack is a flexible package manager that supports multiple versions,
+configurations, platforms, and compilers. This guide provides detailed
+instructions on how to build and install GEOPM using Spack.
+
+Installing Spack
+----------------
+
+First, you need to install Spack. Follow the instructions on the `Spack website
+<https://spack.io>`_ to clone the Spack repository and set up your environment.
+
+.. code-block:: bash
+
+    $ git clone https://github.com/spack/spack.git
+    $ . spack/share/spack/setup-env.sh
+
+Adding GEOPM to Spack
+---------------------
+
+GEOPM is included in the Spack package repository. To add GEOPM to your Spack
+environment, use the following command:
+
+.. code-block:: bash
+
+    $ spack install geopm
```

Comment:
```suggestion
    $ spack install geopm-service
      OR
    $ spack install geopm-runtime
```
"geopm" is the name of the old spack project for releases < 3.0.  They aren't supported, and I need to open a PR on spacks repo to remove them.

Reply:
Every other example invocation below should be one of either geopm-service or geopm-runtime.